sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/ReadingOrder.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+/** Defines values for ReadingOrder. */
+public enum ReadingOrder {
+    /** Enum value basic. */
```

Comment:
it will be good to expand the docstring and explain what each value represents

Reply:
I agree, updated :)